docs/openfaas-cloud/secrets.md

```diff
@@ -0,0 +1,45 @@
+OpenFaaS Cloud **secret** objects lets you store and manage sensitive information securely
+
+
+## Using secrets
+
+Secrets in OpenFaaS Cloud will be mounted as data volumes and made accessible to your function
+
+### Creating secrets
```

Comment:
Define a secret

Reply:
Should this be `Define a sealed secret` following your recommendation to above?